website/src/docs/hotchocolate/v11/fetching-data/dataloader.md

```diff
@@ -40,7 +40,7 @@ A query against the above schema could look like the following:
 
 The above request fetches two persons in one go without the need to call the backend twice. The problem with the GraphQL backend is that field resolvers are atomic and do not have any knowledge about the query as a whole. So, a field resolver does not know that it will be called multiple times in parallel to fetch similar or equal data from the same data source.
 
-The idea of a dataloader is to batch these two requests into one call to the database.
+The idea of a data loader is to batch these two requests into one call to the database.
```

Comment:
These should actually be written as [DataLoader](https://github.com/graphql/dataloader), as it refers to the utility or pattern, and not just to the loading of data in general.

Reply:
Ah, that makes total sense, it just didn't seem correct as written. Fixed!